frontend/apps/app/next.config.ts

```diff
@@ -13,6 +13,10 @@ if (process.env.VERCEL_ENV === 'production') {
 }
 
 const nextConfig: NextConfig = {
+  // Server-only packages that should not be bundled on the client
+  serverExternalPackages: ['@mastra/*'],
+  output: 'standalone',
+
```

Comment:
ref: https://qiita.com/har1101/items/11c913c34f5f8ba088aa#mastra%E3%82%A2%E3%83%97%E3%83%AA%E3%82%92%E4%BD%9C%E6%88%90%E3%81%99%E3%82%8B

Reply:
I understood that these settings are required when deploying in a container and not necessary when deploying to Vercel, but were they necessary?

Reply:
These settings are required even in local development to prevent build errors caused by Mastra's internal file imports.🤔
Without them, Next.js may try to parse unsupported files like `.md` and fail.

![ss 3298](https://github.com/user-attachments/assets/0be5d418-f48a-4ced-b8ee-d1578cb2aece)

Reply:
@FunamaYukina 
I understand the problem. However, the problem is that only `serverExternalPackages` needs to be set, and `output: 'standalone'` should not be necessary. I checked on my local development server and output was not needed.

---

Changing to `output: 'standalone'` will significantly change the output produced by the build. This is not optimized for Vercel and may cause unexpected problems. This is why I do not want to change `output`.
https://nextjs.org/docs/app/api-reference/config/next-config-js/output

Reply:
Oh, sorry, the `output: 'standalone'` setting was not mandatory as you said. I removed it.🙏
https://github.com/liam-hq/liam/pull/1664/commits/7ddba41802707e3a27d69261c6882b19ddac19e7

Reply:
Thanks 👍🏻